build.gradle.kts

```diff
@@ -374,3 +375,7 @@ subprojects {
         }
     }
 }
+
+//tasks.named("check").configure {
+//    dependsOn(":tests:konsist:test")
+//}
```

Comment:
Oops, this is not needed. I'll remove it when I revert the break konsist commit.